apps/meteor/client/views/room/Header/Omnichannel/QuickActions/hooks/useQuickActions.tsx

```diff
@@ -203,13 +203,15 @@ export const useQuickActions = (
 						  }
 						: { transcriptEmail: { sendToVisitor: false } }),
 				});
+				homeRoute.push();
+				RoomManager.close(room.t + rid);
```

Comment:
not really sure how this works, but, is the `l{someroomid}` format valid? no `/` in the concatenation? 👀

Reply:
maybe `RoomManager.close(`${room.t}${rid}`);`?

Reply:
No no, i was referring to, is that `t + id` what the .close function expects? If that's the case, pls ignore me lol

Reply:
Yeah, `t + id` is what is expected 😢